include/he.h

```diff
@@ -335,13 +335,35 @@ typedef he_return_code_t (*he_inside_write_cb_t)(he_conn_t *conn, uint8_t *packe
  * @param length The length of the entire packet in bytes
  * @param context A pointer to the user defined context
  * @see he_conn_set_context Sets the value of the context pointer
+ * @deprecated Use he_outside_write_ex_cb_t instead.
  *
  * Whenever Helium needs to do an outside write this function will be called. On Linux this would
  * usually be writing to a UDP socket to send encrypted data over the Internet.
  */
 typedef he_return_code_t (*he_outside_write_cb_t)(he_conn_t *conn, uint8_t *packet, size_t length,
                                                   void *context);
 
+/// Instruct the outside_write_cb function to set the DF flag on the IP packet
+#define HE_OUTSIDE_WRITE_DONT_FRAGMENT 0x0001
+
+/// Default flags for outside_write_cb
+#define HE_OUTSIDE_WRITE_DEFAULT_FLAGS 0x0000
+
+/**
+ * @brief The prototype for the outside write callback function
+ * @param conn A pointer to the connection that triggered this callback
+ * @param packet A pointer to the packet data
+ * @param length The length of the entire packet in bytes
+ * @param flags A 32-bits field to control the behaviors when doing the outside write function
+ * @param context A pointer to the user defined context
+ * @see he_conn_set_context Sets the value of the context pointer
+ *
+ * Whenever Helium needs to do an outside write this function will be called. On Linux this would
```

Comment:
Any reason to specify `On Linux ` explicitly ? Is it not similar in other platforms too ?

Reply:
Nothing specific, but this comment has been here (outside_write_cb_t) for a long time.